tests/ember_debug/object-inspector-test.js

```diff
@@ -27,6 +26,7 @@ function setupApp() {
 let ignoreErrors = true;
 
 module("Ember Debug - Object Inspector", {
+  // eslint-disable-next-line object-shorthand
   beforeEach: async function() {
```

Comment:
tried that, but my IDE screamed at me when I did it... might not be supported that well yet